_posts/2018-04-17-libra.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Libra
+date: 2018-04-17
+categories:
+    - project
+    - Flask
+    - MySQL
+    - Flask_SQLAlchemy
+    - Python
+description: web knowledgebase where users to input skills and view skills of other users
+image: 
+image-sm: 
+author: Fernando Abascal
+author-image: https://avatars3.githubusercontent.com/u/31594822?s=460&v=4
+author-bio: 3rd Year CIT Student
+author-email: fxa6511@rit.edu
+author-social:
+    github: https://github.com/nandoascal
+    linkedin: 
+---
+Libra was meant to allow CSH members to have a place to congregate their wealth of knowledge into one place. My hopes is that down the line Libra will enable
+members of house to easily track down other members that have particular skillsets and then take that step to learn those skills. Sometimes it could be
+something simple like python that many people on floor understand. Other times it could be some obscure framework that one guy managed to incorporate into
+a peanut dispensing app. Regardless, it is useful having a place where you can track this stuff down.
+
+Libra in its current state allows users to create, edit, and delete their own skills, see a list of all users that have entered skills into the site, and
+search for specific skills.
+
+I loved learning frontend and although I had a lot of trouble, I was able to also learn how to properly communicate and post information to a MySQL database using python.
+
+I do hope to continue to improve the site and add more features until it is a fully fleshed out and much more viable product. Also, my frontend design is really poor presently
```

Comment:
> I do hope to continue to improve the site and add more features until it is a fully fleshed out and much more viable product.

First part of this sentence sounds messy; maybe something like:

"I hope to continue improving the site and adding more features until it is a fully fleshed out and much more viable product."

(Kinda a nitpick, ignore it if you want)